### ISSUE TITLE
Can the author reproduce his own code?

### ISSUE THREAD

**Issue by Nanasaki-Ai** [skelemoa/psumnet#3]
There are a lot of problems in the code.

First, in "Pretrained weights and scores files", the author only shows the model trained by the body stream. The other two streams, i.e., legs and hands, are not shown. The authors show the predicted softmax classification scores (available in the pkl file). Here comes the new problem. Taking ntu60_xsub as an example, the dictionary of each stream only contains the classification scores of 16478 test samples, but in fact there are a total of 16487 test samples.

Second, this work is based on CTR-GCN. The author seems to have only changed the name of the class in model/psumnet.py. OK, after all, the model of multi-stream fusion in advance is still used, and the author also divides graph into three categories for multi-stream fusion. In the data preprocessing stage, CTR-GCN encapsulates training and testing samples and labels in an npz file. It seems that PSUMNet does not act like this. So I wonder how to get the pkl files of these labels.

In psumnet/feeders/feeder_ntu.py, line 34~38:
    if os.path.exists(label_path):
        with open(label_path, 'rb') as f:
            self.pickle_file = pickle.load(f)
            self.sample_name = self.pickle_file[0][0]
            self.label = self.pickle_file[0][1]

In psumnet/config/ntu_configs/body.yaml, line 8 and 22:
    label_path: #/train_label_path/
    test_path: #/train_label_path/

Third, there are so many bugs in this code need to be fixed, e.g., ensemble.py, feeder_ntu.py, etc.

I am very grateful to the author for opening the source code, but also ask the author to be responsible for his own source code. Can the author reproduce his own effects on a new machine only by relying on these open codes?

I don't doubt that the results in the paper can be obtained by decaying the learning rate and setting a large number of epochs. However, the author should also consider whether others can try to reproduce it when opening the code.
